elements/pfe-badge/test/pfe-badge_test.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+    <script src="/components/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="/components/web-component-tester/browser.js"></script>
+    <script type="module" src="../pfe-badge.js"></script>
```

Comment:
@diwanshi - Can you update this to `../dist/pfe-badge.js` so the tests start passing?

Reply:
@kylebuch8 updated.